OloEngine/src/OloEngine/Networking/Replication/EntitySnapshot.cpp

```
@@ -0,0 +1,162 @@
+
+        if (data.empty())
+        {
+            return;
+        }
+
+        FMemoryReader reader(data);
+        reader.ArIsNetArchive = true;
+
+        while (reader.Tell() < reader.TotalSize() && !reader.IsError())
+        {
+            u64 uuid = 0;
```

Clang (14.0.6)

[warning] variable 'uuid' is not initialized

(cppcoreguidelines-init-variables)